packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -96,10 +105,14 @@ MultipathableSrs.propTypes = {
     pgpus: getPgpus,
   }
 })
+@addSubscriptions(() => ({
+  plugins: subscribePlugins,
+}))
 export default class extends Component {
   async componentDidMount() {
     this.setState({
       isHtEnabled: await isHyperThreadingEnabledHost(this.props.host),
+      isNetDataInstalledOnHost: await isNetDataInstalledOnHost(this.props.host),
```

Comment:
You can't call this at all if the plugin is not installed and loaded.

Reply:
@pdonias Why not call this method and handle the error?

But I don't like that it will add an entry to the logs, maybe we should not log the method not found errors?